### DISCUSSION TITLE
Adjusting FOV on UE5.3 Version

### DISCUSSION THREAD

**Discussion by DansTip** [Sixze/ALS-Refactored#367] [Q&A] [answered]
Anyone know how to do this? Just moved over from the Market Place version and its grayed out, still confused on how to adjust the FOV

**Comment by 4re** [comment]


**Reply by DansTip** [reply]
I have no clue how to find that, In the folders Or where? I'm so confused. Idk why he would move the setting

**Reply by DansTip** [reply]
Just went through Visual Studio and have no clue where to find a SIMPLE FOV adjustment

**Comment by DansTip** [comment, marked as answer]
So No thanks to Jimbo, I managed to by clicking random things after scavenging through C++, its actually very simple Here: Plugins/Advanced Locomotion System Refactored Content/ALSCamera/Data/CS_Als_Default. That being said Double click or open the CS_Als_Default Data Asset Highlighted Pink, and type in FOV in the search bar, and bam you got FP FOV and TP FOV. Your welcome to anyone in the future that came across this same issue.
